fapp/src/pages/CreateExpense/CreateExpense.jsx

```diff
@@ -33,6 +34,18 @@ export default function CreateExpense() {
 
   const { currentUser } = useContext(AuthContext);
 
+  const saveFile = (fileToUpload,newId) =>{
+    const storage = getStorage();
```

Comment:
Shouldn't you import fileStorage from backend/firebase here?

Reply:
ok